graphql/tests/query.rs

```diff
@@ -267,7 +267,7 @@ async fn insert_test_entities(
     manifest: SubgraphManifest<graph_chain_ethereum::Chain>,
     id_type: IdType,
 ) -> DeploymentLocator {
-    let deployment = DeploymentCreate::new(&manifest, None);
+    let deployment = DeploymentCreate::new(String::new(), &manifest, None);
```

Comment:
Could we make the raw yaml an `Option` here? Alternatively, `deployment::set_manifest_raw_yaml` could translate an empty string back to `None`.

Reply:
If the tests work, I would prefer not to make the interface more option-y just for the tests.

Reply:
Oh, I totally missed that that was in the tests. Just ignore what I said.